gym_solo/envs/solo8v2vanilla.py

```diff
@@ -80,6 +88,12 @@ def step(self, action: List[float]) -> Tuple[solo_types.obs, float, bool,
         observation, the reward for that step, whether or not the episode 
         terminates, and an info dict for misc diagnostic details.
     """
+    if self._normalize:
+      a = np.array(action)
+      low = self._action_space.low
+      hi = self._action_space.high
+      action = low + ((a + 1) * (hi - low)) / 2
```

Comment:
Same comment about parenthesis to make it easier to read

Reply:
done